### PR TITLE
Jamie Winter Assignment 2

### PR DESCRIPTION
**Pull request by jailarie**
renamed the README and fixed directory structure from Assignment 2 pull request. Hadn't actually pulled from master before resubmitting.

### CODE REVIEW

File: problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (cjyovich) [inline problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java:7]: Maybe add a comment up top indicating what this program is. The class name does indicate that but I still think it doesn't hurt for readability.

File: problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		int sum = 0;
+		
+		while( current < n) {
+			if(current % 3 == 0 || current % 5 == 0) {
+				sum += current;
+			}
+			current++;
+		}System.out.println("The sum of multiples of 3 and 5 below " + n + " is " + sum);
+	}	
+}
```
Comment (cjyovich) [inline problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java:34]: Code looks good! Though the problem specifies to do this operation for numbers under 1000, I think the program could be improved/extended by adding functionality to let the user set the value of n. Would help with unit testing, too.

File: problems/euler/multiplesOf3And5/java/jailarie/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+
+Find the sum of all the multiples of 3 or 5 below 1000.
+
+##Solution
+Loop through each number from 1 to 999. If the number can be divided by 3 or 5 with a remainder of 0 add it to the sum.
+
+##Running the Program
+Open the Java IDE of your choice. (If you do not have a java IDE installed Eclipse can be downloaded or free. Open eulerProblemOne.java in your IDE and run as Java application. Solution will be printed to the console.
```
Comment (cjyovich) [inline problems/euler/multiplesOf3And5/java/jailarie/README.md]: README looks good - the only issue I can find is a tiny spelling error in line 13- "Eclipse can be downloaded _for_ free"

**Review by cjyovich — COMMENTED** [review]
Nice work!! See inline comments for specifics but overall, this looks great. +1

**Review by jjg-akers — APPROVED** [review]
The code looks good and the readme was easy enough to follow. +10
Screen Shot 2020-02-28 at 1 43 01 PM

File: problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java (hunk 1 above)
Comment (jailarie) [reply problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java:7]: Good call. I added a short comment. Thanks!

File: problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java (hunk 2 above)
Comment (jailarie) [reply problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java:34]: I had thought about doing that originally, but was trying to stick to the original problem. Made the change for unit testing. Thanks @cjyovich !

File: problems/euler/multiplesOf3And5/java/jailarie/README.md (hunk 3 above)
Comment (jailarie) [reply problems/euler/multiplesOf3And5/java/jailarie/README.md]: That damn f key! Fixed it. Thanks @cjyovich !

Comment (cjyovich) [reply problems/euler/multiplesOf3And5/java/jailarie/README.md]: Claaaaassic F key!

File: problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java (hunk 2 above)
Comment (cjyovich) [reply problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java:34]: Nice!

**Pull request merged**